tests/cli/upload/test_cli_upload_observations.py

```diff
@@ -0,0 +1,137 @@
+"""Test cg.cli.upload.observations module."""
+
+import logging
+
+import pytest
+from _pytest.logging import LogCaptureFixture
+from click.testing import CliRunner
+
+from cg.apps.loqus import LoqusdbAPI
+from cg.cli.upload.observations import observations
+from cg.cli.upload.observations.utils import (
+    get_observations_case,
+    get_observations_case_to_upload,
+    get_observations_api,
+    get_observations_case_to_delete,
+)
+from cg.constants import EXIT_SUCCESS
+from cg.constants.subject import Gender
+from cg.exc import CaseNotFoundError
+from cg.meta.upload.observations.observations_api import UploadObservationsAPI
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+from tests.store_helpers import StoreHelpers
+
+from tests.apps.loqus.conftest import (
+    fixture_loqusdbapi,
+    fixture_loqus_config,
+    loqus_process,
+    loqus_binary_path,
+    loqus_config_path,
+    fixture_loqusdb_case_output,
+)
+
+
+def test_observations(
+    base_context: CGConfig, cli_runner: CliRunner, helpers: StoreHelpers, caplog: LogCaptureFixture
+):
+    """Test upload of observations."""
+
+    caplog.set_level(logging.DEBUG)
+    store: Store = base_context.status_db
+
+    # GIVEN a case ready to be uploaded to Loqusdb
+    case: models.Family = helpers.add_case(store)
+    case.customer.loqus_upload = True
+    sample: models.Sample = helpers.add_sample(store, application_type="wes")
+    store.relate_sample(family=case, sample=sample, status=Gender.UNKNOWN)
```

Comment:
```suggestion
    store.relate_sample(family=case, sample=sample, status=PhenotypeStatus.UNKNOWN)
```
My bad. This should be correct for the relate table. Please correct throughout